### PR TITLE
Changing data saving to dataframes and parameters

### PR DESCRIPTION
**Pull request by GuillermoAbadLopez**
THIS PR TACKLES 0), 1) and 3)  of the following points, together with https://github.com/qilimanjaro-tech/qililab-portfolio/pull/39 tackling 0), 1) and 2)

0) Change `platform_params` into `platform_parameters` ✅ 

1) Create a python script that runs a linear calibration for 1 qubit ✅  

2) Create a bash script for linear calibration ✅

3) How will the data be saved? ✅

> - Fidelities and set_parameters into dataframes
> - Change `platform_params` to `platform_parameters`

### CODE REVIEW

**Comment by GuillermoAbadLopez** [comment]
This is how its going:
Screenshot 2024-01-03 at 21 22 05

**Comment by GuillermoAbadLopez** [comment]
Improved one:
Screenshot 2024-01-04 at 00 55 18

**Review by jjmartinezQT — CHANGES_REQUESTED** [review]
Can we merge main into this to incorporate latest changes from main, fix the unittest and the code quality before further reviewing? Thanks!

**Comment by GuillermoAbadLopez** [comment]
> Can we merge main into this to incorporate latest changes from main, fix the unittest and the code quality before further reviewing? Thanks!

Yep definitely, sorry its not passing because I did some changes for everything to work together better! I just put it in InProgress, and I tackle what you said today! 🙌

**Review by jjmartinezQT — CHANGES_REQUESTED** [review]
This review is still showing the set_parameter changes that are already merged in main no?

File: src/qililab/calibration/calibration_controller.py [hunk 1]
```diff
@@ -572,18 +598,30 @@ def get_last_set_parameters(self) -> dict[tuple, tuple]:
             if (
                 node.output_parameters is not None
                 and node.previous_timestamp is not None
-                and "platform_params" in node.output_parameters
+                and "platform_parameters" in node.output_parameters
             ):
-                for bus, qubit, parameter, value in node.output_parameters["platform_params"]:
+                for bus, qubit, parameter, value in node.output_parameters["platform_parameters"]:
+                    if qubit is None:
+                        qubits = [int(s) for s in re.findall(r"\d+", "he33llo 42 I'm a 32 string 30")]
```
[the remark this answers is not in the document]
Comment (GuillermoAbadLopez) [reply src/qililab/calibration/calibration_controller.py:605]: Very nice catch haha

**Comment by GuillermoAbadLopez** [comment]
Last version of the tables:
Screenshot 2024-01-05 at 19 06 03

**Review by jjmartinezQT — CHANGES_REQUESTED** [review]
An unittest for calibrate_all is needed, which Isaac is already taking care of. Aside from that, I will test this one running a full autocalibration once the chip is online again in slurm and then we will be able to merge it.

**Review by jjmartinezQT — APPROVED** [review]
This is good to go, amazing job.

**Pull request merged**